_partials/_prometheus-integrate.md

```diff
@@ -2,74 +2,167 @@ import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.md
+   
+   1. Name your exporter. 
+
+   1. Change the auto-generated Prometheus credentials, if needed. See [official documentation][prometheus-authentication] on basic authentication in Prometheus. 
+
+      ![Prometheus exporter credentials](https://assets.timescale.com/docs/images/prometheus-authentication.png)
+
+1. **Attach the exporter to a $SERVICE_SHORT**
+
+   1. Select a $SERVICE_SHORT, then click `Operations` > `Exporters`. 
 
-   1. Grant the `pg_read_all_stats` permission to the `monitoring` user:
+   1. Select the exporter in the drop-down, then click `Attach exporter`.
 
-      ```sql
-      GRANT pg_read_all_stats to monitoring;
+      ![Attach a Prometheus exporter to a Timescale Cloud service](https://assets.timescale.com/docs/images/attach-prometheus-exporter-timescale-cloud.png)
+
+   The exporter is now attached to your $SERVICE_SHORT. You can unattach it by clicking the trash bin icon in the exporter list. 
+
+      ![Unattach a Prometheus exporter from a Timescale Cloud service](https://assets.timescale.com/docs/images/unattach-prometheus-exporter-timescale-service.png)
+
+1. **Configure the Prometheus scrape target**
+
+   1. Select your service > `Operations` > `Exporters` and click the information icon next to the exporter. You see the exporter details. 
+
+      ![Prometheus exporter details in Timescale Cloud](https://assets.timescale.com/docs/images/prometheus-exporter-details.png)
+
+   1. Copy the exporter URL. 
+
+   1. In your Prometheus installation, update `prometheus.yml` to point to the exporter URL as a scrape target:
+
+      ```yml
+      scrape_configs:
+       - job_name: "timescaledb-exporter"
+         scheme: https
+         static_configs:
+           - targets: ["my-exporter-url"]
+         basic_auth:
+           username: "user"
+           password: "pass"
       ```
+   
+      See [Prometheus documentation][scrape-targets] for details on configuring scrape targets. 
+
+      You can now monitor your $SERVICE_SHORT metrics. Use the following metrics to check the service is running correctly:
+
+      *   `timescale.cloud.system.cpu.usage.millicores`
+      *   `timescale.cloud.system.cpu.total.millicores`
+      *   `timescale.cloud.system.memory.usage.bytes`
+      *   `timescale.cloud.system.memory.total.bytes`
+      *   `timescale.cloud.system.disk.usage.bytes`
+      *   `timescale.cloud.system.disk.total.bytes`
+
+      Additionally, use the following tags to filter your results.
+
+      |Tag|Example variable| Description                |
+      |-|-|----------------------------|
+      |`host`|`us-east-1.timescale.cloud`|                            |
+      |`project-id`||                            |
+      |`service-id`||                            |
+      |`region`|`us-east-1`| AWS region                 |
+      |`role`|`replica` or `primary`| For $SERVICE_SHORT with replicas |
+   
+</Procedure>
+
+
+
+</Tab>
+
+<Tab title="Self-hosted TimescaleDB">
```

Comment:
Strictly speaking it's also applicable for a cloud prometheus, but customer managed prometheus exporter